src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -88,7 +90,7 @@ const resolveLandingPageLinks = (
     //   href: '#FIXME-need-link',
     //   text: 'Download my IRS 1095-B form',
     // },
-  ];
+  ].filter(isLinkData);
```

Comment:
I think this could be `.filter(link => !!link)` since the line `registered && { href, link }` will have been processed to either be a link or `null/undefined/false`.

Reply:
Yeah, in the existing scenarios the conditional expressions should resolve to `false`, but wanted to be ready for other situations such as if an expression yielded an empty object.